catalogs/catalog-kafka/src/test/java/com/datastrato/gravitino/catalog/kafka/integration/CatalogKafkaIT.java

```diff
@@ -0,0 +1,370 @@
+import static com.datastrato.gravitino.catalog.kafka.KafkaTopicPropertiesMetadata.REPLICATION_FACTOR;
+import static com.datastrato.gravitino.integration.test.container.KafkaContainer.DEFAULT_BROKER_PORT;
+
+import com.datastrato.gravitino.Catalog;
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.Namespace;
+import com.datastrato.gravitino.client.GravitinoMetalake;
+import com.datastrato.gravitino.dto.messaging.TopicDTO;
+import com.datastrato.gravitino.integration.test.container.ContainerSuite;
+import com.datastrato.gravitino.integration.test.util.AbstractIT;
+import com.datastrato.gravitino.integration.test.util.GravitinoITUtils;
+import com.datastrato.gravitino.messaging.Topic;
+import com.datastrato.gravitino.messaging.TopicChange;
+import com.datastrato.gravitino.rel.Schema;
+import com.datastrato.gravitino.rel.SchemaChange;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.Config;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.config.ConfigResource;
+import org.apache.kafka.common.config.TopicConfig;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Tag("gravitino-docker-it")
+public class CatalogKafkaIT extends AbstractIT {
+  private static final Logger LOG = LoggerFactory.getLogger(CatalogKafkaIT.class);
+  private static final ContainerSuite CONTAINER_SUITE = ContainerSuite.getInstance();
+  private static final String METALAKE_NAME =
+      GravitinoITUtils.genRandomName("catalogKafkaIT_metalake");
+  private static final String CATALOG_NAME =
+      GravitinoITUtils.genRandomName("catalogKafkaIT_catalog");
+  private static final String DEFAULT_SCHEMA_NAME = "default";
+  private static final String PROVIDER = "kafka";
+  private static GravitinoMetalake METALAKE;
+  private static Catalog CATALOG;
```

Comment:
Generally, static final fields should be in uppercase.

Reply:
yeah, it's all caps. What's the problem?

Reply:
private static Catalog CATALOG are not final